debian/libxpum-dev.install

```diff
@@ -1,3 +1,3 @@
 usr/lib/xpu-smi
```

Comment:
@androniychuk do you know ?

Reply:
I'm open to removing those, but I've kept them in since they appear to be test resources, and I don't want to package tests that will fail without necessary resources. @androniychuk are these important for the package?

Reply:
I did not mean to remove them, just they do not seem to be in the right package, we have to check but they can be useful to `xpu-smi` or even `libxxpum1`

Reply:
everything in this dir, needs to be there
"usr/lib/xpu-smi"

Reply:
here's some relevant docs

https://github.com/intel/xpumanager/blob/master/doc/smi_user_guide.md#diagnose-gpu-with-different-test-suites

Reply:
@mckees my feeling is that we need to install this folder for the `xpu-smi` package, but we should double check

Reply:
I just realized that this is in the -dev pkg...
I totally agree it must be in xpu-smi pkg

Reply:
Alright, I switched it to the xpu-smi package. Since it was under libs but wasn't part of the actual libs, I decided to put it in -dev